addons/base/views.py

```diff
@@ -65,6 +65,7 @@
 
 # import so that associated listener is instantiated and gets emails
 from website.notifications.events.files import FileEvent  # noqa
+from osf.utils.requests import requests_retry_session
```

Comment:
Can we move this import to after `from osf.utils import permissions` in line 59?